src/production/stages/codeSplittingStage.ts

```diff
@@ -50,7 +50,7 @@ export function codeSplittingStage(props: IProductionFlow) {
 
       const bundle = createBundle({
         ctx: props.ctx,
-        name: `${useHash ? fastHash(root.module.props.absPath) + '-' : ''}${beautifyBundleName(root.module.props.absPath, maxlen)}`,
+        name: `${useHash ? fastHash(root.module.props.absPath) + '-' : ''}${beautifyBundleName(root.module.props.fuseBoxPath, maxlen)}`,
```

Comment:
`fuseBoxPath` is a nicer reference that doesn't include the larger absolute path. This gives us "hash like" structure to reduce filename collision.

Reply:
I think you should include pkg.getPublicName which might Include version. Or at least check if there are external deps (basically you are able to lazy load a library - that's my concern)

Reply:
absPath does this but fuseBoxPath doesn’t?

Are you saying for the case 


Import(‘https://lodash.com”)

?

Reply:
Do you have a test-case example?